### PR TITLE
Using correct URI path, using correct params

### PR DESCRIPTION
**Pull request by cassiascheffer**
I had made two mistakes in my previous work. The URI should be singular,
and `account_holder` needs the `bank_` prefix.

### CODE REVIEW

File: lib/bambora/bank/payment_profile_resource.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def initialize(client:, api_key:, version: DEFAULT_VERSION)
         @client = client
         @api_key = api_key
         @version = version
-        @sub_path = '/scripts/payment_profiles.asp'
+        @sub_path = '/scripts/payment_profile.asp'
```
Comment (cassiascheffer) [inline lib/bambora/bank/payment_profile_resource.rb:33]: Changed this to singular.

File: lib/bambora/bank/payment_profile_resource.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def initialize(client:, api_key:, version: DEFAULT_VERSION)
       #   data = {
       #     customer_code: '1234',
       #     bank_account_type: 'CA',
-      #     account_holder: 'All-Maudra Mayrin',
+      #     bank_account_holder: 'All-Maudra Mayrin',
```
Comment (cassiascheffer) [inline lib/bambora/bank/payment_profile_resource.rb:43]: Updated docs to show that this needs the `bank_` prefix.

File: lib/bambora/rest/www_form_client.rb [hunk 3]
```diff
@@ -19,8 +19,8 @@ def post(path:, body:)
         # did not work in testing so I am manually transforming the hash into query parameters here.
         parse_response_body(
           super(
-            path: "#{path}?#{Bambora::Builders::WWWFormParameters.new(body: body)}",
-            body: nil,
+            path: path,
+            body: Bambora::Builders::WWWFormParameters.new(body: body).to_s,
```
Comment (cassiascheffer) [inline lib/bambora/rest/www_form_client.rb:23]: Moved this to the body as per the example in the bambora docs. Although, it works in both places just fine.

**Review by cassiascheffer — COMMENTED** [review]
ᕦ( ✿ ⊙ ͜ʖ ⊙ ✿ )━☆ﾟ.*･｡ﾟ

File: lib/bambora/rest/client.rb [hunk 4]
```diff
@@ -29,6 +29,7 @@ def get(path:, params:, headers:)
       end
 
       def post(path:, body:, headers:)
+        require 'pry'; binding.pry
```
Comment (harrylewis) [inline lib/bambora/rest/client.rb]: Should this be removed?

**Review by harrylewis — CHANGES_REQUESTED** [review]
Changes look good, and it sounds like you have tested it all with our tests accounts. Marking as "request changes" because of `binding.pry` that is still there. I'll approve once that's removed.

File: lib/bambora/client.rb [hunk 5]
```diff
@@ -154,7 +155,7 @@ def json_client
 
     def www_form_client
       @www_form_client ||= Bambora::Rest::WWWFormClient.new(
-        base_url: base_url,
+        base_url: scripts_api_base_url,
```
Comment (cassiascheffer) [inline lib/bambora/client.rb:158]: Added a second URL in here since the legacy stuff has a different base url.

**Review by harrylewis — APPROVED** [review]
🚀

**Pull request merged**